### PR TITLE
add option to edit node config

### PR DESCRIPTION
**Pull request by jormeli**
If `default.json` is available for a node, opens a configuration form with field values pre-populated from it.

### CODE REVIEW

File: webui/src/Components/ConfigurationForm.tsx [hunk 1]
```diff
@@ -141,9 +142,13 @@ const AttrsOfControl = (props: AttrsOfControlProps) => {
                 <FormControl mr={4} mb={4}>
                   <FormLabel>{key}</FormLabel>
                   {typeof value == 'boolean' ? (
-                    <CustomCheckbox name={jp.stringify([...keys, item, key])} checked={value} />
+                    <CustomCheckbox name={jp.stringify([...keys, item, key])} isChecked={value || false} />
```
Comment (jhvst) [inline webui/src/Components/ConfigurationForm.tsx:145]: `isChecked={value || false}`

Do I want to know why the `|| false` is necessary?

**Review by jhvst — APPROVED** [review]
LGTM

File: webui/src/Components/ConfigurationForm.tsx (hunk 1 above)
Comment (jormeli) [reply webui/src/Components/ConfigurationForm.tsx:145]: I lost faith in life while fighting the typechecker

Comment (jhvst) [reply webui/src/Components/ConfigurationForm.tsx:145]: I premused.

Carry on.

**Pull request merged**